src/utils/Processing.ts

```diff
@@ -42,13 +42,19 @@ export async function getResources(
 
 export function writeFSH(resources: Package, outDir: string): void {
   const exporter = new FSHExporter(resources);
-  const outputPath = path.join(outDir, 'resources.fsh');
-  fs.writeFileSync(outputPath, exporter.export());
-  logger.info(`Wrote fsh to ${outputPath}.`);
-  if (resources.configuration) {
-    const configPath = path.join(outDir, 'config.yaml');
-    fs.writeFileSync(configPath, resources.configuration.toFSH());
-    logger.info(`Wrote config to ${configPath}.`);
+  try {
+    const resourceDir = ensureOutputDir(path.join(outDir, 'input', 'fsh'));
```

Comment:
I think instead of using `ensureOutputDir` here, we might want to just use `ensureDirSync` directly. The problem with `ensureOutputDir` is that it logs a message indicating the output directory, so since we call `ensureOutputDir` once in `app.ts`, and now once here, we get a somewhat confusing combination of log messages:
![image](https://user-images.githubusercontent.com/52747882/99970210-86201e80-2d69-11eb-8bd2-3827d7cc6c82.png)